platform-includes/session-replay/setup-canvas/javascript.angular.mdx

```diff
@@ -0,0 +1,33 @@
+```javascript
+import * as Sentry from "@sentry/angular-ivy";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+
+  integrations: [
+    // Keep the Replay integration as before
+    Sentry.replayIntegration(),
+
+    // The following is all you need to enable canvas recording with Replay
+    Sentry.replayCanvasIntegration(),
+  ],
+});
+```
+
+```html {tabTitle: CDN}
```

Comment:
We can (and should) remove the CDN/Loader snippets for everything that is not browser!